install.bash

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [[ "$1" == "inchina" ]]; then
+  export WE_ARE_IN_CHINA=true
+fi
+
+echo
+echo "Welcome to SQLFlow playground!"
+echo
+
+if ! which vagrant >/dev/null; then
+  echo "Install vagrant first, please refer to https://www.vagrantup.com/downloads.html"
+  echo
+  echo "If you are using macOS, the operation system may prevent you from installing the web-downloaded package, you can try use brew to install vagrant. like:"
+  echo "brew cask install vagrant"
+  echo
+  echo "Fix this and re-run this script again please!"
+  exit 0
+fi
+
+if [[ -n "$(vagrant global-status --prune | grep -o 'playground')" ]]; then
+  echo "It seems you have already installed our playground, exiting..."
+  exit 0
+fi
+
+if [[ "$WE_ARE_IN_CHINA" ]]; then
```

Comment:
How do we know that we are in China? And, users in other regions around the world might also need to use their local mirrors.  How about we don't care if users are specifically in China, but support mirrors by downloading with `axel` other than `wget`?

Reply:
I did tried to use `axel --search`, it turned out that the axel relied `filesearching.com` can't be accessed in China.
I also want to  give multi urls to axels, but I't seems we can't enumerate the box's mirrors, really, vagrant do not have so many well-known official mirrors like other software.

Plus the fact we have to consider how to install axel itself in kinds of OS, I decided to make it work first, just use the wget. 

Especially for China, I add a param `inchina`. Actually, if this param is not present, the script just run `vagrant up` and delegate the work to standard vagrant provision process.
```
./install.bash inchina
```

Other thought:
I want to make the install more intelligent, so I tried to figure out if  we are in China automatically. I found [this tool](https://unixmen.com/find-fastest-mirror-debian-derivatives/). I will try it in a more standard environment (in the vm, we don't have to consider the os). It will in my next PR.